lib/node_modules/@stdlib/ndarray/base/some-by/lib/0d.js

```diff
@@ -0,0 +1,84 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Tests whether at least `n` elements in an ndarray pass a test implemented by a predicate function.
+*
+* @private
+* @param {Object} x - object containing ndarray meta data
+* @param {ndarrayLike} x.ref - reference to the original ndarray-like object
+* @param {string} x.dtype - data type
+* @param {Collection} x.data - data buffer
+* @param {NonNegativeIntegerArray} x.shape - dimensions
+* @param {IntegerArray} x.strides - stride lengths
+* @param {NonNegativeInteger} x.offset - index offset
+* @param {string} x.order - specifies whether `x` is row-major (C-style) or column-major (Fortran-style)
+* @param {PositiveInteger} n - number of elements
+* @param {Function} predicate - predicate function
+* @param {*} thisArg - predicate function execution context
+* @returns {boolean} boolean indicating whether all elements pass a test
```

Comment:
This description is not correct more generally. This is not `every-by`. Should be

```suggestion
* @returns {boolean} boolean indicating whether at least `n` elements pass a test
```

This applies here and everywhere else in this PR.